README.md

```diff
@@ -1,124 +1,18 @@
-﻿# snet-cli
+# snet-python-monorepo
 
 [![CircleCI](https://circleci.com/gh/singnet/snet-cli.svg?style=svg)](https://circleci.com/gh/singnet/snet-cli)
   
-SingularityNET CLI
+SingularityNET Python Monorepo
 
-## Getting Started  
+## Packages  
+This repository is a monorepo that includes several packages that we publish to PyPI from a shared codebase, specifically:
   
-These instructions are for the development and use of the SingularityNET CLI.
-For further details, please check our full [Documentation](http://snet-cli-docs.singularitynet.io/).
+|Package                                       |Description                                                          |
```

Comment:
Lets also link the readme of the cli and SDK here, makes it easier for people to navigate